common/djangoapps/entitlements/api/v1/serializers.py

```diff
@@ -24,3 +29,40 @@ class Meta:
             'mode',
             'order_number'
         )
+
+
+class CourseEntitlementSupportDetailSerializer(serializers.ModelSerializer):
+    """ Serialize the details of a support team interaction with a learner's course entitlement. """
+    support_user = serializers.SlugRelatedField(
```

Comment:
with this being read_only / a slugrelated field, we cant use this serializer for posting/updating right? would modifying this to not be readonly mean we could use the serializer for creation as well?

Reply:
I don't actually know. Is that something you'd want to be able to do, to use the serializer for creation?

Reply:
Maybe not? I know that in ecommerce the product end point has a bunch of readonly stuff that made us unable to generalize the endpoint for posts. Though this is a more niche endpoint, so its probably ok for now.